crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -370,6 +370,32 @@ def target_platform(options = {})
         end
       end
 
+      # Check if they are any instances running.
+      # It's necessary for disruptive upgrade, when user has to shut them down manually before
+      # proceeding with the services shutdown.
+      def check_for_running_instances
```

Comment:
We'll need to add some check for this into our jobs - suspend running VM's before the services step. Including the manila one @Itxaka mentioned.